src/views/iam/AddLdapAccount.vue

```diff
@@ -283,7 +283,7 @@ export default {
       params.domainid = store.getters.userInfo.domainid
       if (domain) {
         const result = this.listDomains.filter(item => item.name === domain)
-        if (result) {
+        if (result.length > 0) {
```

Comment:
how about result == None?

Reply:
Can't be none. Either an empty list of not